plugins/azure/eventGrid/domainMinimumTlsVersion.js

```diff
@@ -0,0 +1,71 @@
+var async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Event Grid Domain Minimum TLS Version',
+    category: 'Event Grid',
+    domain: 'Management and Governance',
+    severity: 'Medium',
+    description: 'Ensures that Azure Event Grid domain is using the latest TLS version.',
+    more_info: 'Using latest TLS version for Event Grid domains enforces strict security measures, which requires that clients send and receive data with a newer version of TLS.',
+    recommended_action: 'Modify Event Grid domain to set the desired minimum TLS version.',
+    link: 'https://learn.microsoft.com/en-us/azure/event-grid/transport-layer-security-configure-minimum-version',
+    apis: ['eventGrid:listDomains'],
+    settings: {
+        event_grid_domain_min_tls_version: {
```

Comment:
this shouldn't be a setting, we should determine which is the latest stable version

Reply:
we have similar plugin for event Hub, since the previous versions are still supported in console. so maybe we can add setting that customer can set for its desired tls version